code/utils/utils_c.c

```diff
@@ -0,0 +1,31 @@
+//
+// Created by Fábio Domingues on 19/07/17.
+//
+
+#include "utils_c.h"
+#include <linux/blk_types.h>
+
+bool __c_is_async_write(struct disk_write_op_meta *m) {
```

Comment:
Identifiers starting with `_` are generally "reserved" by the compiler or the language. Could you please change these to start with something else?

(SO further reading [here](https://stackoverflow.com/questions/25090635/use-and-in-c-programs) if you so desire)